src/XMakeTasks/UnitTests/GenerateResource_Tests.cs

```diff
@@ -419,7 +417,7 @@ public void ForceSomeOutOfDate()
                 DateTime time2 = File.GetLastWriteTime(t.OutputResources[1].ItemSpec);
 
                 System.Threading.Thread.Sleep(200);
-                if (NativeMethodsShared.IsOSX)
+                if (!NativeMethodsShared.IsWindows)
```

Comment:
Technically we shouldn't be relying on the file system to give us timestamps , the code should be mock-able so that we're testing its logic in a more deterministic way.  But until we have our tests set up like that, I'll just update the comment 😢